tool/gen_mackerel_plugin.pl

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env perl
```

Comment:
I added test code to Makefile to detect missing `go generate` execution as following.


```shell
test `go generate ./... && git diff | wc -l` = 0 || (echo 'please `go generate ./...` and commit them' && exit 1)
```